### DISCUSSION TITLE
How to suppress / Hide System Default Message on API

### DISCUSSION THREAD

**Discussion by Stephen-Lobo** [chatwoot/chatwoot#2628] [Q&A]
Is there any way to hide / not to display the default system messages when a new Conversation / Label / Message created via API ?

Maybe passing 'true/false' option in the API code. 

<img width="682" alt="System messages" src="https://user-images.githubusercontent.com/87435610/125649962-0438c91b-4c26-4fc4-b9c4-3fabb84869ef.png">

**Comment by sojan-official** [comment]
We don't have a way at the moment, But i have created a new issue for the same https://github.com/chatwoot/chatwoot/issues/2667

**Comment by Stephen-Lobo** [comment]
Thank you.
